### DISCUSSION TITLE
[Feature] Add 'Show Faces' Option When Naming an Unidentified Person.

### DISCUSSION THREAD

**Discussion by shumancw** [immich-app/immich#14080] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When identifying a grouping of unnamed faces, add a menu option to view a thumbnail view of the person identified.

**Issue:** Especially in family photos where there can be a lot of similarities between people, there are many time when the photos presented contain two people that look similar and the model chooses the wrong person.

**Example:** 10 photos are presented containing a group of people. The person I _think_ is the person identified in all 10 photos is actually only identified correctly in 8 of the photos. In 2 of the photos presented it actually identified a cousin as being the same as the person in the other 8 photos.

**Feature:** Give the ability for the 10 pictures of the unnamed person to show as thumbnails or zoomed in version of the person identified. Doing this allow for a user to quickly confirm that 8 of the photos belong to one person and 2 to another.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Is this a duplicate of #6013?

**Reply by shumancw** [reply]
Potentially. I was thinking specifically of workflow issues when working with unnamed people, but assuming #6013 was included in the 'unnamed' view, then it would be a dup.

Edit: Looks like the search term I missed was "facebox".
